CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Console errors: `tagLists` on `ProjectCard` and `items`, `key`, `tag`
```

Comment:
Ce serait plus clair avec simplement "Fix console warnings from `ProjectCard` and `LinkList` rendering".